packages/fiori/test/pages/DialogWithBar.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+	<meta charset="utf-8">
+
+	<title>Dialog test page</title>
+	<meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no">
+	<meta charset="utf-8">
+
+	<script src="../../bundle.esm.js" type="module"></script>
+</head>
+
+<body class="bar1auto">
+	<ui5-button id="dialog-bar-open">Dialog with Bar in Footer</ui5-button>
+	<ui5-dialog open id="dialog-bar" header-text="Using bar with design=Footer as footer">
+		<ui5-bar slot="footer" design="Footer">
```

Comment:
this is in a separate test page at
[http://localhost:8080/packages/**fiori**/test/pages/DialogWithBar.html](http://localhost:8080/packages/fiori/test/pages/DialogWithBar.html)
 because
[http://localhost:8080/packages/**main**/test/pages/Dialog.html](http://localhost:8080/packages/main/test/pages/Dialog.html) doesn't have access to the fiori package